### DISCUSSION TITLE
Issues to install UI of Phoebe in  MAC

### DISCUSSION THREAD

**Discussion by mdredyanahmed** [phoebe-project/phoebe2-ui#20] [Questions]
Can anyone help me to install or execute UI in mac?

I have phoebe installed in macbook pro m1 via pip but I am unable to launch UI for this

**Comment by kecnry** [comment]
The UI is installed separately (via one of the [release builds](https://github.com/phoebe-project/phoebe2-ui/releases/tag/1.0.1) or manually from this repo via the instructions in the readme - although manual building can be difficult), or you can use the web UI at ui.phoebe-project.org (the hosted server is currently down, but if you launch `phoebe-server` locally, you should be able to attach to your own running instance via localhost (`localhost:5555` would be the default location, see `phoebe-server --help` for more options).

**Comment by mdredyanahmed** [comment]
I tried to download the UI from release but I didn't find any compatible version for macbook pro m1. Any suggestion in this regard?

Moreover, after installing phoebe, I tried to launch UI by phoebe-server, it doesn't work.
